Components/Sources/Components/Components/Editors/Source Editor/WKSourceEditorTextFrameworkMediator.swift

```diff
@@ -13,6 +13,15 @@ fileprivate var needsTextKit2: Bool {
     }
 }
 
+class WKSourceEditorTextView: UITextView {
+    override func accessibilityActivate() -> Bool {
+
+        UIAccessibility.post(notification: .announcement, argument: "Loading editor text.")
```

Comment:
Patterning usage elsewhere, should we localize this argument string?

Reply:
👍 Good catch!